src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastCache.cs

```diff
@@ -250,7 +250,7 @@ internal static CastResult TryGet(int[] table, nuint source, nuint target)
             TableMask(ref tableData) = size - 1;
 
             // Fibonacci hash reduces the value into desired range by shifting right by the number of leading zeroes in 'size-1'
-            byte shift = (byte)BitOperations.LeadingZeroCount(size - 1);
```

Comment:
That's super subtle. @tannergooding, why is there a signed overload for Int128 but no signed overloads for sbyte, short, int, or long?